Client/Application/AppDelegate+Experiments.swift

```diff
@@ -10,8 +10,33 @@ import Foundation
 
 extension AppDelegate {
     func initializeExperiments() {
-        let options = Experiments.getLocalExperimentData()
-            .map(Experiments.InitializationOptions.testing(localPayload:)) ?? .normal
+
+        let defaults = UserDefaults.standard
+        let nimbusFirstRun = "NimbusFirstRun"
+        let firstRun = defaults.object(forKey: nimbusFirstRun) != nil
+        defaults.set(false, forKey: nimbusFirstRun)
+
+        let initialExperiments = Bundle.main.url(forResource: "initial_experiments", withExtension: "json")
```

Comment:
I am guessing this is temporary 😅

Reply:
This is the basis of first-startup and first-startup-after-upgrade experiments. The file itself will change, and perhaps not checked into the git, but that this code is trying to load the file, this is as designed.